app/controllers/case_contacts/form_controller.rb

```diff
@@ -65,6 +64,17 @@ def get_cases_and_contact_types
     @selected_contact_type_ids = @case_contact.contact_type_ids
   end
 
+  def manage_navigation
+    if @nav_step.present?
+      jump_to(@nav_step.split("/").last.to_sym)
```

Comment:
I'm confused what @nav_step is... why not make it correspond to a wizard step? Is it pulling double duty as a step and as a path?